.github/workflows/csharp-qltest.yml

```diff
@@ -64,20 +64,35 @@ jobs:
           codeql test run --threads=0 --ram 50000 --slice ${{ matrix.slice }} --search-path extractor-pack --check-databases --check-undefined-labels --check-repeated-labels --check-redefined-labels --consistency-queries ql/consistency-queries ql/test --compilation-cache "${{ steps.query-cache.outputs.cache-dir }}"
         env:
           GITHUB_TOKEN: ${{ github.token }}
-  unit-tests:
+  unit-tests-linux:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
       - name: Setup dotnet
         uses: actions/setup-dotnet@v3
         with:
           dotnet-version: 7.0.102
-      - name: Extractor unit tests
+      - name: Extractor unit tests (Linux)
         run: |
           dotnet test -p:RuntimeFrameworkVersion=7.0.2 extractor/Semmle.Util.Tests
           dotnet test -p:RuntimeFrameworkVersion=7.0.2 extractor/Semmle.Extraction.Tests
           dotnet test -p:RuntimeFrameworkVersion=7.0.2 autobuilder/Semmle.Autobuild.CSharp.Tests
           dotnet test -p:RuntimeFrameworkVersion=7.0.2 "${{ github.workspace }}/cpp/autobuilder/Semmle.Autobuild.Cpp.Tests"
+  unit-tests-windows:
```

Comment:
I think you may be able to use a matrix (containing `ubuntu-latest` and `windows-2019`) to reduce duplication.

Reply:
Yes, that is a good idea (TIL). My skills for defining workflows are limited. As far as google informs me the default shell for all other types of runner than windows runners is *bash*, so setting the default to *bash* for non-windows test executions should also be fine.

Reply:
Rewrite looks great!